pkg/json_canonical/json_canonical_test.go

```diff
@@ -0,0 +1,185 @@
+package json_canonical_test
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/atlantistechnology/sdt/pkg/json_canonical"
+	"github.com/atlantistechnology/sdt/pkg/types"
+)
+
+type File struct {
+	name   string
+	digest string
+}
+
+func testHash(filename string, digest string) bool {
```

Comment:
What is the purpose here? It looks like a test function, but doesn't take a *testing.T parameter? Assuming it's valid, can this have a comment added?

Reply:
As per Slack comment, this is a utility function not a test function.  However, it is duplicated across many tests, so I'll factor it out into `utils` instead anyway.